BankTransferApp/Program.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+
+namespace BankTransfers
+{
+    class Program
+    {
+        private Bank _bank;
+        private UserInterface _userInterface;
+
+        private void Run()
+        {
+            _userInterface = new UserInterface();
+            _bank = new Bank();
+
+            do
+            {
+                _userInterface.DisplayMenu();
+                var selectedMenuOption = _userInterface.ReadMenu();
+
+                switch (selectedMenuOption)
+                {
+                    case 1:
+                        CreateAccount();
+                        break;
+                    case 2:
+                        DomesticTransfer();
+                        break;
+                    case 3:
+                        OutgoingTransfer();
+                        break;
+                    case 4:
+                        ListAccountsBalance();
+                        break;
+                    case 5:
+                        ListTransfers();
+                        break;
+                    case 6:
+                        Environment.Exit(0);
+                        break;
+                    default:
+                        Console.WriteLine("Unknown menu option");
+                        break;
+                }
+            } while (true);
+        }
+
+        private void CreateAccount()
+        {
+            _userInterface.DisplayCreateAccountInfo();
+            var accountName = _userInterface.GetAccountName();
+            var account = _bank.CreateAccount(accountName);
+            Console.WriteLine("Creating new account");
+        }
+
+        private void DomesticTransfer()
```

Comment:
Domestic transfer and outgoing transfer methods have quite a lot in common. Common parts can be extracted to a separate method.